nrf_wifi/fw_if/umac_if/inc/fmac_api_common.h

```diff
@@ -241,6 +241,53 @@ enum nrf_wifi_status nrf_wifi_fmac_get_reg(struct nrf_wifi_fmac_dev_ctx *fmac_de
 enum nrf_wifi_status nrf_wifi_fmac_get_power_save_info(void *fmac_dev_ctx,
 						       unsigned char if_idx);
 
+#ifdef CONFIG_NRF700X_RAWDATA_TX
+/**
+ * @brief Get or set the current mode of operation
+ * @param dev_ctx Pointer to the UMAC IF context for a RPU WLAN device.
+ * @param if_idx Index of the interface on which mode is to be set.
+ * @param mode value to be set for the interface.
+ *
+ * This function is used to send a command
+ *         to RPU to Set/Get mode of operation
+ *
+ * @return Command execution status
```

Comment:
This needs to be consistent everywhere. E,g, for `nrf_wifi_fmac_start_rawpkt_xmit` we have a different description and a different one here. (Please check for all newly added functions in this PR).

Reply:
updated as discussed